cypress/integration/scriptSpec.js

```diff
@@ -0,0 +1,56 @@
+import config from '../support/config';
+
+[
+  { service: 'news', url: `/news/articles/${config.assets.news}` },
+  {
+    service: 'persian',
+    url: `/persian/articles/${config.assets.persian}`,
+  },
+  { service: 'igbo', url: `/igbo` },
+  { service: 'yoruba', url: `/yoruba` },
+  { service: 'pidgin', url: `/pidgin` },
+].forEach(({ service, url }) => {
+  describe(`Script src - ${service}`, () => {
+    beforeEach(() => {
+      cy.visit(url);
+    });
+
+    it('should only have expected bundle script tags', () => {
+      cy.onLocal(() => {
+        cy.get('script[src]').each($p =>
```

Comment:
Why is this variable name-spaced with `$`?

Reply:
Convention I guess, e.g. https://docs.cypress.io/api/commands/each.html#Usage

Reply:
Based on [this](https://docs.cypress.io/api/commands/each.html#DOM-Elements) it seems to be related to getting the DOM in a jQuery fashion, feels weird using dollar notation as it's usually JQuery or PHP 

>// $el is a wrapped JQuery element

Reply:
It is Jquery though